lib/commands.js

```diff
@@ -37,6 +57,52 @@ export default class Commands {
   toggleActiveEditor() {
     this.emitter.emit('should-toggle-active-editor')
   }
+  showDebug(standardLinters: Array<Linter>, indieLinters: Array<IndieDelegate>, uiProviders: Array<UI>) {
+    if (!manifest) {
+      manifest = require('../package.json')
+    }
+    if (!Helpers) {
+      Helpers = require('./helpers')
+    }
+
+    const textEditor = atom.workspace.getActiveTextEditor()
+    const textEditorScopes = Helpers.getEditorCursorScopes(textEditor)
+    const sortedLinters = standardLinters.slice().sort(sortByName)
+    const sortedIndieLinters = indieLinters.slice().sort(sortByName)
+    const sortedUIProviders = uiProviders.slice().sort(sortByName)
+
+    const indieLinterNames = sortedIndieLinters
+      .map(formatItem).join('\n')
+    const standardLinterNames = sortedLinters
+      .map(formatItem).join('\n')
+    const matchingStandardLinters = sortedLinters
+      .filter(linter => Helpers.shouldTriggerLinter(linter, false, textEditorScopes))
+      .map(formatItem).join('\n')
+    const humanizedScopes = textEditorScopes
+      .map(formatItem).join('\n')
+    const uiProviderNames = sortedUIProviders
+      .map(formatItem).join('\n')
+
+    const ignoreGlob = atom.config.get('linter.ignoreGlob')
+    const disabledLinters = atom.config.get('linter.disabledProviders')
+      .map(formatItem).join('\n')
+
+    atom.notifications.addInfo('Linter Debug Info', {
+      detail: [
+        `Platform: ${process.platform}`,
+        `Atom Version: ${atom.getVersion()}`,
+        `Linter Version: ${manifest.version}`,
+        `Ignore Glob: ${ignoreGlob}`,
+        `UI Providers: \n${uiProviderNames}`,
+        `Indie Linter Providers: \n${indieLinterNames}`,
+        `Standard Linter Providers: \n${standardLinterNames}`,
+        `Matching Linter Providers: \n${matchingStandardLinters}`,
```

Comment:
Can this one and the Disabled ones be put up where the lists start (L97)? These are the two important lists so they should be the most visible.